src/api/app/controllers/concerns/webui/requests_filter.rb

```diff
@@ -80,4 +80,8 @@ def filter_by_text(text)
   def staging_projects
     Project.where(name: @filter_staging_projects)
   end
+
+  def allowed_involvements
```

Comment:
Doing so, we can override the allowed involvements in the "concerned" controllers